README.md

```diff
@@ -7,67 +7,70 @@
+- Dramatically simplify the development and deployment process.
 
-- scaffold your dapp project
-- ease the development and deployment process
-- create custom task for blockchain and contract interaction with less boilerplate code
-  - using terra.js directly could be cumbersome, Terrain gives you extensions and utilities to help you
-- console to terra blockchain, an interactive repl which have the same extensions and utilities as custom task
-- ability to predefine functions to be used in task and console
+Terrain is **not**:
 
-[![oclif](https://img.shields.io/badge/cli-oclif-brightgreen.svg)](https://oclif.io)
-[![Version](https://img.shields.io/npm/v/terrain.svg)](https://npmjs.org/package/terrain)
-[![Downloads/week](https://img.shields.io/npm/dw/terrain.svg)](https://npmjs.org/package/terrain)
+- A fully-featured Terra command-line interface (CLI). For this, take a look at <a href="https://docs.terra.money/docs/develop/how-to/terrad/using-terrad.html" target="_blank">terrad</a>.
+- A Light Client Daemon (LCD). You will still need an RPC endpoint to deploy your contract. <a href="https://github.com/terra-money/LocalTerra#readme" target="_blank">LocalTerra</a> is the recommended option for this.
+
+---
+
+# Table of contents
 
 <!-- toc -->
-* [Terrain](#terrain)
-* [Setup](#setup)
-* [Getting Started](#getting-started)
-* [Migrating CosmWasm contracts on Terra](#migrating-cosmwasm-contracts-on-terra)
-* [Usage](#usage)
-* [Commands](#commands)
-* [Use main branch in local.](#use-main-branch-in-local)
+
+- [Setup](#setup)
+- [Getting Started](#getting-started)
+- [Migrating CosmWasm Contracts on Terra](#migrating-cosmwasm-contracts-on-terra)
+- [Usage](#usage)
+- [Commands](#commands)
+- [Use Terrain Main Branch Locally](#use-terrain-main-branch-locally)
 <!-- tocstop -->
 
 # Setup
 
 ## Download LocalTerra
 
-For local developement environment, you need [LocalTerra](https://github.com/terra-money/localterra).
+For testing purposes, we recommend to install and run LocalTerra on your personal computer. Instructions on how to get LocalTerra up and running may be found in the <a href="https://github.com/terra-money/LocalTerra#readme" target="_blank">LocalTerra documentation</a>.
 
-_**note:** if you are using m1 chip, you might need to update your Docker Desktop due to [qemu bug](https://github.com/docker/for-mac/issues/5561)_
+To summarize, once all dependencies are installed, you may then clone the LocalTerra repo, change into the new directory and spin up an instance of the environment utilizing Docker.
 
-```sh
-git clone --branch v0.5.17 --depth 1 https://github.com/terra-money/localterra
-cd localterra
+```
+git clone https://github.com/terra-money/LocalTerra.git
+cd LocalTerra
 docker-compose up
 ```
 
-## Setup Rust
+<sub>**Note:** _If you are using a Mac with an M1 chip, you might need to update your Docker Desktop due to the <a href="https://github.com/docker/for-mac/issues/5561" target="_blank">qemu bug</a>._</sub>
```

Comment:
Can we try to find a different way to represent this kind of texts? Because it looks very small to me, what do you think?